src/block-components/helpers/size/style.js

```diff
@@ -53,6 +53,20 @@ const Styles = props => {
 				responsive="all"
 				hasUnits="px"
 			/>
+			<BlockCss
+				{ ...propsToPass }
+				selector={ wrapperSelector || selector }
+				styleRule="width"
+				attrName="width"
+				key="width"
+				attrNameTemplate={ attrNameTemplate }
+				responsive="all"
+				hasUnits="px"
+				valueCallback={ ( value, getAttribute ) => {
+					const blockWidth = getAttribute( 'width' )
+					return blockWidth ? 'inherit' : ''
```

Comment:
Please use `undefined` here, I think there's a chance that the style might show up if it's an empty string.
```suggestion
					return blockWidth ? 'inherit' : undefined
```